components/contribution-flow/index.js

```diff
@@ -831,6 +844,14 @@ class ContributionFlow extends React.Component {
                     disabledPaymentMethodTypes={this.props.disabledPaymentMethodTypes}
                     hideCreditCardPostalCode={this.props.hideCreditCardPostalCode}
                   />
+                  {isCaptchaEnabled() && currentStep.name === STEPS.PAYMENT && !LoggedInUser && (
```

Comment:
The last step of the crypto flow seems different. Might be more future-proof to just check that we're on the last step:

```suggestion
                  {isCaptchaEnabled() && !nextStep && !LoggedInUser && (
```